middleware/router.go

```diff
@@ -53,9 +53,15 @@ func NewRouter(router *gin.Engine) *gin.Engine {
 
 	// HTML Template generated pages
 	router.GET("/status", httpTemplates.StatusPageResponse(), StatusPageMiddleware())
+	router.GET("/login", LoginPageMiddleware())
+
+	// Authentication
+	router.POST("/sign-up", api.SignUp)
+	router.POST("/sign-in", api.SignIn)
+	router.GET("/sign-out", api.SignOut)
```

Comment:
You decided not to change it?
https://github.com/LookinLabs/GoBlitz/pull/22#discussion_r1623393431